src/theory/quantifiers/sygus/sygus_unif_strat.h

```diff
@@ -387,13 +398,34 @@ class SygusUnifStrategy
    *
    * This method builds the mapping needs_cons, which maps (master) enumerators
    * to a map from the constructors that it needs.
+   *
+   * It also may directly add simple symmetry breaking lemmas to strategy_lemmas
```

Comment:
Shouldnt these be symmetry breaking *templates*?  It looks like the code is only excluding top-level occurrences.